experimental/LieAlgebras/src/CoxeterGroup.jl

```diff
@@ -1,4 +1,4 @@
-abstract type CoxeterGroup end # <: Group
+abstract type CoxeterGroup end
```

Comment:
I think we should remove CoxeterGroup for now, as there is no content anyway (here and in the exports)